### ISSUE TITLE
[HOLD for payment 2023-05-22] [$1000] Avatars are not shown on bill split preview while offline

### ISSUE THREAD

**Issue by Julesssss** [Expensify/App#18472]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
- Locate a group chat with multiple users
- Go offline
- Create a bill split

## Expected Result:
- The avatars should be displayed

## Actual Result:
- The avatars are not displayed until you go online

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [x] iOS / native
- [x] iOS / Safari
- [x] MacOS / Chrome / Safari
- [x] MacOS / Desktop

**Version Number:** Dev build
**Reproducible in staging?:** Yes (once [this PR](https://github.com/Expensify/App/pull/18143) is deployed)
**Reproducible in production?:** No

![Screenshot 2023-05-05 at 12 23 12](https://user-images.githubusercontent.com/10736861/236445243-cf39df25-dd14-4f49-9acf-60a4fcebeb14.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0114b795aa8cbc7508</li>
        <li>Upwork Job ID: 1655924184745324544</li>
        <li>Last Price Increase: 2023-05-09</li>
    </ul>
</details>

**Comment by Julesssss** [comment]
We shouldn't open this issue up for contributors until [this PR](https://github.com/Expensify/App/pull/18143) is deployed.

**Comment by stephanieelliott** [comment]
> We shouldn't open this issue up for contributors until https://github.com/Expensify/App/pull/18143 is deployed.

Sounds good, PR is still under review so will hold on applying any labels.

**Comment by Julesssss** [comment]
THanks @stephanieelliott. We merged the PR, so please feel free to apply the label now

**Comment by Julesssss** [comment]
Ah, good spot. Given how important this feature is I don't think we should care too much about the order for now. I'd be happy to treat that as a follow-up issue.

**Comment by Julesssss** [comment]
@Prince-Mendiratta I created a[ follow-up issue](https://github.com/Expensify/App/issues/18641) for that, so don't worry about fixing that.

**Comment by s77rt** [comment]
Regarding the order, we have this issue that may be similar https://github.com/Expensify/App/issues/15644

**Comment by Julesssss** [comment]
Yeah, I think you're right. But we're going to introduce a new instance of it. Also, we need to expand the number shown, so I think it's fine to keep as a separate issue.

**Comment by stephanieelliott** [comment]
PR is undergoing QA!

**Comment by Julesssss** [comment]
Now merged. Awaiting deployment and payment...

**Comment by s77rt** [comment]
- The PR that introduced the bug has been identified: Not a regression but a missing case that could have been handled on https://github.com/Expensify/App/pull/11597
- The offending PR has been commented on: https://github.com/Expensify/App/pull/11597#issuecomment-1547587977
- A discussion in #expensify-bugs has been started: https://expensify.slack.com/archives/C049HHMV9SM/p1684146206357539
- Determine if we should create a regression test for this bug: Yes

___

**Regression Test Proposal**

1. Open App
2. Turn off Internet Connection
3. Split bill with any 2 users (Click on the FAB > Split bill and follow the process)
4. Verify that the group chat has the IOU preview and it's greyed out
5. Verify that user avatars are shown in the IOU preview

Test should probably be added to [C1971182](https://expensify.testrail.io/index.php?/cases/view/1971182).

**Comment by Julesssss** [comment]
Hey @stephanieelliott, I think this one is ready for payment.

**Comment by stephanieelliott** [comment]
Upwork job offered to @Prince-Mendiratta and @s77rt, please accept when you have a sec and I can issue payment!

**Comment by s77rt** [comment]
Accepted!
